### DISCUSSION TITLE
can't get it to work in pycharm

### DISCUSSION THREAD

**Discussion by McPhail42** [nutti/fake-bpy-module#291] [Q&A]
i installed via pip in my venv, added the venv to my sources root, added paths etc, but the from . import statements seem not to work for intellisense. package bmesh is recognized, but bpy only gets generictype1 and 2. ops for example still gets a yellow underline with an error when i hover over it: cannot find reference 'ops' in '__init__.pyi'

**Comment by nutti** [comment]
This issue is now solved by #304.
